src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/processors/BlobProcessor.java

```diff
@@ -105,13 +105,17 @@ private void processBlob(String blobName, String containerName) {
                         envelopeId
                     );
                 } else {
-                    logger.warn("Invalid signature. Blob name: {} container: {}", blobName, containerName);
-                    envelopeRepository.insert(createNewEnvelope(
+                    logger.error("Invalid signature. Blob name: {} container: {}", blobName, containerName);
+                    UUID envelopeId = envelopeRepository.insert(createNewEnvelope(
                         blobName,
                         containerName,
                         blobCreationDate,
                         Status.REJECTED
                     ));
+
+                    logger.info(
+                        "Rejected Blob. Name {} container {}. Envelope ID: {}", blobName, containerName, envelopeId
+                    );
```

Comment:
perhaps we can combine into one log entry? not sure how precious are super edge case scenario people but failing to insert should rarely happen

Reply:
updated